pkg/registrationserver/registrationserver.go

```diff
@@ -0,0 +1,126 @@
+package registrationserver
```

Comment:
I would suggest to use `server` as the pkg name, and `server.go` as the "main" file in this pkg. The repo is `registration-service`, so I believe there's no need to duplicate the `registration` term